src/AAVE/Modal/SupplyModal.jsx

```diff
@@ -307,7 +319,22 @@ return (
                       props={{
                         left: <GrayTexture>${state.amountInUSD}</GrayTexture>,
                         right: (
-                          <GrayTexture>Wallet Balance: {balance}</GrayTexture>
+                          <GrayTexture>
+                            Wallet Balance: {balance}
+                            <Max
+                              onClick={() => {
+                                changeValue(
+                                  symbol === "ETH" || symbol === "WETH"
```

Comment:
I suggest we move the special handling for ETH/WETH into `changeValue()` function.